Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[[source]]
+url = "https://pypi.pacificclimate.org/simple"
+verify_ssl = true
+name = "pcic"
+
+[packages]
+pywps = ">=4.2.6"
+nchelpers = "==5.5.7"
+netCDF4 = ">=1.5.4"
+beautifulsoup4 = "==4.9.3"
+
+[requires]
+python_version = "3.8"
+
+[dev-packages]
+pytest = "==5.4.3"
+black = "==19.10b0"
```

Comment:
This is perfect!

Reply:
Only note is that I don't see `rpy2` in here. We should include it in the general install when using `pipenv`.